core/files/file-types/svg.php

```diff
@@ -290,7 +290,7 @@ public static function get_inline_svg( $attachment_id ) {
 
 		$attachment_file = get_attached_file( $attachment_id );
 
-		if ( ! $attachment_file ) {
+		if ( ! $attachment_file || ! file_exists( $attachment_file ) ) {
```

Comment:
Why not just
```suggestion
		if ( ! file_exists( $attachment_file ) ) {
```
?

Reply:
@udidol it makes sense! I just changed the code.